### DISCUSSION TITLE
No Wi-Fi after flashing D1 mini second time

### DISCUSSION THREAD

**Discussion by Moosbueffel** [arendst/Tasmota#11915] [Q&A] [answered]
I read FAQ and troubleshooting, searching internet, but have no idea at all.  It is a new D1 mini, only used before to test functionality with a little program with blinking LED. Than I found, that I can use Tasmota for this device also :-). But than ... :-(

First I flashed the current release V9.4.0 tasmota-DE.bin. All seems fine. I got the local WiFi to enter my data. I used the found SSID, which was correct (MeineBox). But saving, I got always the error, that SSID/password is wrong. But I'm sure It was right. I tried it three times. 

In the meantime I setup a POW with Tasmota. This without any problems to connect to my WiFi. Than I thought - ok WiFi is ok, Tasmota V9.x is ok, lets flash again. And now I didn't get anything. No local network (192.168.4.1) for entering my data! 

I also tried the RESET with 40sec pushing the RESET Button. But nothing. After the 40 sec, when I leave the button, the LED flashes on time. Thats all. 

My flashing should be well and I thougt, that flashing will reset all to firmware derfaults!?

![grafik](https://user-images.githubusercontent.com/7726457/116259251-efbbb300-a775-11eb-8436-ca04d70e0384.png)

**Comment by sfromis** [comment, marked as answer]
A good debugging technique is to try resetting the device with a serial terminal program listening to the USB/serial adapter. The console output will tell you if/what Tasmota is doing.

**Reply by Moosbueffel** [reply]
With Arduino IDE I got the following:
![grafik](https://user-images.githubusercontent.com/7726457/116286351-6add9300-a78f-11eb-821a-859d7220a17e.png)

Does it help to help me?

**Reply by sfromis** [reply]
Those messages indicate a completely normal initial boot waiting for you to connect to the AP, and do initial configuration. Until configured, it will reboot every 3 minutes.

**Reply by Moosbueffel** [reply]
Magic !? I now did the same as before, but not from my old PC. I tried it from my smartphone (to enter my credentials for my network). 

**Now it works :-).**

I have now idea, what happend what is different now. But I'm glad.
